### ISSUE TITLE
Uncaught Error: prompt() is and will not be supported

### ISSUE THREAD

**Issue by al-yakubovich** [skandasoft/browser-plus#182]
"prompt" doesn't work at all in browse-plus. When I call window.prompt I get “Error: prompt() is and will not be supported” in a console. How can I fix it?

**Comment by williamschneider** [comment]
I agree. I was trying to figure this out too. If this can never be supported, could you at least let us know why? Is the functionality too difficult to work in this plug-in? Cause I was trying to do something for a class and they were using Brackets which could do this. I wanna brag on Atom as being superior really...
